site/data/s3.ts

```diff
@@ -54,3 +54,27 @@ export const getItem = async (bucket: string, key: string, originalFilename: str
         throw error;
     }
 };
+
+export const getObject = async (bucket: string, key: string, originalFilename: string): Promise<Uint8Array | null> => {
+    logger.info("", {
+        context: "data.s3",
+        message: "getting item from s3",
+    });
+
+    try {
+        const input = {
+            Bucket: bucket,
+            Key: key,
+            ResponseContentDisposition: 'attachment; filename ="' + originalFilename + '"',
+        };
+        const command = new GetObjectCommand(input);
+        const response = await s3.send(command);
+        return response.Body?.transformToByteArray() ?? null;
```

Comment:
Could we return response.Body and so that the method `transformToByteArray` is invoked at the client end?

Reply:
If in future we need to read files from S3 then we'll have to make changes to tested code

Reply:
as discussed